libs/ui/src/components/MediaItem/MediaItem.vue

```diff
@@ -1,5 +1,5 @@
 <template>
```

Comment:
this file is used only for preview ui, we use the one in `components/basic/MediaItem`